client_manager test function

sl5035 opened the pull request.

sl5035 committed:
client_manager test function init

sbillinge commented:
Hi @sl5035 I changed the CI, added codecov and moved from circleci to GH actions.  Please can you update your local client branch, and then merge it into this branch so the PR is using the new CI?  THnaks so much.

sbillinge commented:
btw, I removed ruamel.yaml dependency.  I think it is better to just have one yaml dependency.  It probably means that fsclient and/or mongoclient is broken but can  be fixed by changing the yaml calls to use pyyaml.  If there is functionality in ruamel.yml that you need, maybe we can change the dependency so it is used everywhere and again we just have one yaml dependency.

sl5035 committed:
Merge branch 'client' into client_manager_test

sl5035 committed:
client_manager test fixed

sl5035 commented:
> Hi @sl5035 I changed the CI, added codecov and moved from circleci to GH actions. Please can you update your local client branch, and then merge it into this branch so the PR is using the new CI? THnaks so much.

Done!

> btw, I removed ruamel.yaml dependency. I think it is better to just have one yaml dependency. It probably means that fsclient and/or mongoclient is broken but can be fixed by changing the yaml calls to use pyyaml. If there is functionality in ruamel.yml that you need, maybe we can change the dependency so it is used everywhere and again we just have one yaml dependency.

Ah, got it. Then I'll be working on fsclient to fix yaml dependencies first.
